scripts/cli/tests/cli_helper_test.py

```diff
@@ -71,3 +74,17 @@ def test_coalesce_nothing() -> None:
     single_item = ""
     coalesced_list = coalesce_multi_single(multi_items, single_item)
     assert coalesced_list == []
+
+
+def test_valid_date_empty_string() -> None:
```

Comment:
You can also create [doctests](https://docs.python.org/3/library/doctest.html) for this kind of unit tests.